CIScripts/TestIntegration.ps1

```diff
@@ -46,6 +46,8 @@ $TestConfiguration = [TestConfiguration] @{
     VMSwitchName = "Layered " + $Env:ADAPTER_NAME;
     VHostName = "vEthernet (HNSTransparent)"
     ForwardingExtensionName = $Env:FORWARDING_EXTENSION_NAME;
+    AgentConfigFilePath = "C:\Program Files\Juniper Networks\Agent\contrail-vrouter-agent.conf";
+    AgentSampleConfigFilePath = "C:\Program Files\Juniper Networks\Agent\contrail-vrouter-agent.conf.sample";
```

Comment:
Not sure if we should move this to env variables.. I guess this is fine until we discuss where we want to keep config files with Raja anyways.